examples/simple-example/simple-example-camunda-8/pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>message-adapter-c8</artifactId>
             <version>${project.version}</version>
```

Comment:
the @EnableZeebeClient Annotation can be removed in the *Application.java because it is deprecated